scripts/news/__main__.py

```diff
@@ -0,0 +1,141 @@
+from collections import defaultdict
+from datetime import datetime, timezone
+from typing import Optional
+
+import click
+
+from . import __version__
+from .constants import *
+from .utils import *
+
+
+@click.group(context_settings=dict(help_option_names=["-h", "--help"]), invoke_without_command=True)
+@click.version_option(version=__version__)
+@click.pass_context
+def cli_main(ctx: click.Context) -> None:
+    """
+    Modmail News 📜🤖.
+
+    As part of discord-modmail’s workflow, any non-trivial change to the codebase
```

Comment:
my ide was flagging the character here, please use a `'` after discord-modmail

Reply:
I have no idea how that popped in. Will do